dist/pinny.js

```diff
@@ -106,13 +110,17 @@
 
                 // only run lockup if another pinny isn't
                 // open and locked the viewport up already
-                !this._activePinnies() && this.$pinny.lockup('lock');
+                if (this._activePinnies()) {
+                    this.$pinny.data('lockup')._trigger('locked');
+                } else {
+                    this.$pinny.lockup('lock');
+                }
 
                 EventPolyfill.on(events.resize, function() {
                     plugin._repaint();
                 });
 
-                if (!this._activePinnies() && iOS7orBelow) {
+                if (!this._activePinnies()) {
```

Comment:
We noticed more issues with not closing the keyboard on iOS 8 so I would +1 this change!